docs/_snippets/webhook-signature-validation.mdx

```diff
@@ -0,0 +1,17 @@
+```typescript
+
+const timestamp = req.headers["x-metriport-timestamp"];
```

Comment:
Our WH requests already have this on the body, we could use that, right?

See how Stripe does it, uses only have to retrieve the signature from the request: https://stripe.com/docs/webhooks#verify-official-libraries

Reply:
Yah that makes sense. That would mean getting rid of the timestamp entirely? If its already in the request.body then its redundant to use it twice